dev/PushNotifications/PushNotificationsLongRunningTask/platform.cpp

```diff
@@ -76,7 +78,12 @@ STDMETHODIMP_(HRESULT __stdcall) NotificationsLongRunningPlatformImpl::RegisterL
     auto lock = m_lock.lock_shared();
     THROW_HR_IF(WPN_E_PLATFORM_UNAVAILABLE, m_shutdown);
 
-    const std::wstring appId = GetAppIdentifier(processName);
+    const std::wstring appId{ GetAppIdentifier(processName) };
+    if (appId.empty())
```

Comment:
Add a comment here that NotificationsLongRunningPlatformImpl::RegisterFullTrustApplication should always be called before this or we silently ignore the API call

Reply:
Here and everywhere else where we have this check.